src/zap.conf

```diff
@@ -48,7 +48,7 @@
 10038	OUTOFSCOPE	http://app:8080/public/img/.*
 10038	OUTOFSCOPE	http://app:8080/public/css/.*
 10038	OUTOFSCOPE	http://app:8080/public/js/.*
-10038	OUTOFSCOPE	http://app:8080/(robots.txt|sitemap.xml|TODO)
+10038	OUTOFSCOPE	http://app:8080/(robots.txt|sitemap.xml|TODO|edit/)
```

Comment:
**Suggestion (non-blocking)**: I'm uncertain where this `edit/` is coming from, I don't see it in the template. Would it be possible to make it not be 404 or else to use `TODO` for the url instead?